fvm/meter/weighted_meter.go

```diff
@@ -324,6 +336,12 @@ type WeightedMeter struct {
 
 	computationIntensities MeteredComputationIntensities
 	memoryIntensities      MeteredMemoryIntensities
+
+	storageUpdateSizeMap     map[StorageInteractionKey]uint64
+	storageReadCounter       uint64
```

Comment:
I think we can drop the read / write counter. if I remember correctly, we don't use it anywhere.

Reply:
discussed offline with Janez. Yes we can remove these counters from code. They don't seem to be useful at least in the foreseeable future.